setup.py

```diff
@@ -274,8 +274,9 @@ def run(self):
       platforms=['any'],
       packages=get_packages(),
       package_data={LIBNAME: get_package_data(LIBNAME, EXTLIST),
-                    'spyderplugins':
-                    get_package_data('spyderplugins', EXTLIST)},
+                    'spyplugins': get_package_data('spyplugins', EXTLIST),
+                    },
+      include_package_data=True,
```

Comment:
I think the wheels was not working without but I tried several things so that might not be needed at all :-)